<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecoverPassword</class>
 <widget class="QWidget" name="RecoverPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>185</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: #1B262C;
}

QLineEdit{
background-color: rgb(18, 26, 30);
color: white;
border: 2px solid gray;
border-radius: 5px;}

QLineEdit:hover{
border: 2px solid rgb(174, 173, 172);
}

QLineEdit:focus{
border: 2px solid rgb(7,134,193)}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="recoveryLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: red;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="recoveryEmail">
       <property name="placeholderText">
        <string>Account Email</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="recoverCancelBtn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
 background-color: #3282B8;
 color: rgb(228,240,237);
 border-style: outset;
 border-width: 2px;
 border-radius: 10px;
 border-color: black;
 min-width: 10em;
 padding: 6px;}

QPushButton:hover{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:focus{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:pressed {
 background-color: #0F4C75;
 border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="recoverBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
 background-color: #0F4C75;
 color: darkgray;
 border-style: outset;
 border-width: 2px;
 border-radius: 10px;
 border-color: black;
 min-width: 10em;
 padding: 6px;}

QPushButton:hover{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:focus{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:pressed {
 background-color: #0F4C75;
 border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>Recover</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
